README.md

```diff
@@ -16,8 +16,10 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
-```
+Helps to contain images within an element so it does not float which happens when the image is taller than the element containing it.
+.clearfix {
+  overflow: auto;
+}
 
```

Comment:
Not quite. When we assign a value of `position: float` to an element, the non-floated elements around it don't "see" that the floated element has height, which leads to the overflow problem your describe. Your example is great, though!